eu.jumplink.Easy6502.json

```diff
@@ -0,0 +1,58 @@
+{
+  "id": "eu.jumplink.Easy6502",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "48",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.node22",
+    "org.freedesktop.Sdk.Extension.typescript"
+  ],
+  "build-options": {
+    "append-path": "/usr/lib/sdk/node22/bin:/usr/lib/sdk/typescript/bin:/app/bin"
+  },
+  "command": "eu.jumplink.Easy6502",
+  "finish-args": [
+    "--device=dri",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--env=GJS_DISABLE_JIT=1"
+  ],
+  "cleanup": [
+    "/include",
+    "/lib/pkgconfig",
+    "/man",
+    "/share/doc",
+    "/share/gtk-doc",
+    "/share/man",
+    "/share/pkgconfig",
+    "*.la",
+    "*.a"
+  ],
+  "modules": [
+    {
+      "name": "blueprint-compiler",
```

Comment:
this module is missing the cleanup rule as per blueprint documentation.

Reply:
I adopted the rule as it is given here: https://github.com/flathub/re.sonny.Junction/blob/master/re.sonny.Junction.json#L36